Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -274,7 +274,7 @@ private void NetworkerDisconnected(NetWorker sender)
 			MasterServerNetworker = null;
 		}
 
-		public void UpdateMasterServerListing(NetWorker server, string comment = null, string gameType = null, string mode = null)
+		public void UpdateMasterServerListing(NetWorker server, string player = null,bool removeplayer = false, string comment = null, string gameType = null, string mode = null)
```

Comment:
new optional parameters should be added to the end of the signature of this method to not cause breaks in exsiting code.